pandas/core/arrays/integer.py

```diff
@@ -261,6 +264,10 @@ def __init__(self, values, mask, copy=False):
     def _from_sequence(cls, scalars, dtype=None, copy=False):
         return integer_array(scalars, dtype=dtype, copy=copy)
 
+    @classmethod
+    def _from_sequence_of_strings(cls, strings, dtype=None, copy=False):
+        return cls._from_sequence([int(x) for x in strings], dtype, copy)
```

Comment:
this is not going to work, what if you have a NaN? I think instead you need to call
``pd.to_numeric(errors='raise')`` on this, then pass to cls._from_sequence